lib/ClangImporter/ImportDecl.cpp

```diff
@@ -816,6 +816,31 @@ static bool isPrintLikeMethod(DeclName name, const DeclContext *dc) {
 using MirroredMethodEntry =
   std::tuple<const clang::ObjCMethodDecl*, ProtocolDecl*, bool /*isAsync*/>;
 
+ImportedType tryImportOptionsTypeForField(const clang::QualType type,
+                                          ClangImporter::Implementation &Impl) {
+  ImportedType importedType;
+  auto fieldType = type;
```

Comment:
Nit: could we just drop const from the parameter?

Reply:
Sure thing! If you have any other comments please do post them. I will try and post a pr to 5.9 soon.